doc/ledctl.pod

```diff
@@ -293,92 +293,115 @@ I<failure> or I<disk_failed> is translated to I<ses_fault>
 
 =back
 
-=head2 List of Devices
+=head1 MODES AND MODE SPECIFIC OPTIONS
```

Comment:
It would be great if you would add, for each mode, an example usage in the Examples section.